netboot-oci-artifacts.md

```diff
@@ -1,44 +1,66 @@
-# Packaging Network Boot files using Image manifest
+# Distributing network boot files via image manifest
 
-Netboot files (e.g. PXE) can be packaged as [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
-using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
-and stored in OCI registry.
+Provisioning bootable containers on bare metal systems require OS installer and bootloader files to be distributed along. This specification provides a standard way to encode and describe such payload.
```

Comment:
can you wrap lines to 100 characters, please

Reply:
The whole document, or this particular line only?

Reply:
whole doc